src/index.html

```diff
@@ -11,7 +11,213 @@
+              id="date"
+              name="date"
+            >
+          </label>
+
+          <label for="agr" class="field-form_without-margin">
+            I accept the term of the agreement
+
+            <input
+              type="checkbox"
+              id="agr"
+              name="agr"
+              required
+            >
+          </label>
+
+        </fieldset>
+      </div>
+
+      <div class="block">
+        <fieldset>
+          <legend>Registration</legend>
+
+          <label for="email" class="field-form">
+            E-mail:
+
+            <input
+              type="email"
+              id="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+          <label for="password" class="field-form_without-margin">
+            Password:
+
+            <input
+              type="password"
+              id="password"
+              name="password"
+              required
+              minlength="8"
+              maxlength="32"
+            >
+          </label>
+
+        </fieldset>
+      </div>
+
+      <div class="block">
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+
+          <div class="field-form">
+            Do you love cats?
+
+            <label for="yes">
+              <input
+                type="radio"
+                id="yes"
+                name="cats"
+                value="yes"
+              >
+              Yes
+            </label>
+            <label for="no">
+              <input
+                type="radio"
+                id="no"
+                name="cats"
+                value="no"
+              >
+              No
+            </label>
+          </div>
+
+          <label for="favorite-color" class="field-form">
+            What is your favorite color?
+
+            <input
+              type="color"
+              id="favorite-color"
+              name="favorite-color"
+            >
+          </label>
+
+          <label for="bed" class="field-form">
+            What time do you go to bed?
+
+            <input
+              type="time"
+              id="bed"
+              name="bed"
+            >
+          </label>
+
+          <label for="cars" class="field-form">
+            What are your favorite brand of cars?
+
+            <select
+              name="cars"
+              required
+              size="4"
+              multiple
+              id="cars"
+            >
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+
+          </label>
+
+          <label for="work" class="field-form_without-margin">
+            How do you rate your work?
+
+            <input
+              type="range"
+              id="work"
+              max="100"
+              value="0"
+              name="work"
+              min="0"
```

Comment:
```suggestion
              min="0"
              max="100"
              value="0"
              name="work"
```
Try to follow some logical order in your attributes. E.g. `min` and `max` should be followed one by another. It's hard to find `min` and `max` values in your code if they are situated separately